sites/mosul/configs/openmrs/initializer_config/liquibase/liquibase.xml

```diff
@@ -89,13 +109,22 @@
             <!-- Only running this to remove previously recorded obs -->
             DELETE FROM obs WHERE value_coded = (SELECT concept_id FROM concept WHERE uuid = '80945AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA');
 
+            DELETE FROM drug WHERE concept_id = (SELECT concept_id FROM concept WHERE uuid = '80945AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA');
```

Comment:
For the drug concept to be removed, the entry from the drug table must be removed, which was skipped in the PR making this change.